go/dhcp/ping.go

```diff
@@ -0,0 +1,170 @@
+package main
```

Comment:
Could you extract this into its own little library so that we can reuse this piece of code in other go processes

Since its in the main package now, nothing else than this package can use it